mlflow/__init__.py

```diff
@@ -43,7 +43,6 @@
 import mlflow.projects as projects
 import mlflow.tracking as tracking
 import mlflow.models
-import mlflow.artifacts
```

Comment:
was this a typo?

Reply:
Not a typo. I thought it was not unused import but I was wrong. I am reverting this change. I am facing some import issues and will update the PR with more changes.